tests/hooks/test_http_hook.py

```diff
@@ -0,0 +1,260 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import unittest
+
+import json
+
+import requests
+import requests_mock
+
+import tenacity
+
+from airflow import configuration, models
+from airflow.exceptions import AirflowException
+
+try:
+    from airflow.hooks.http_hook import HttpHook
```

Comment:
This should always be importable

Reply:
I copied these from core - happy to change the behaviour though

Reply:
Please